src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -593,6 +620,22 @@ SHAMapStoreImp::clearPrior (LedgerIndex lastRotated)
     if (health())
         return;
 
+    clearSql (*ledgerDb_,
+        "DELETE FROM Validations WHERE LedgerHash IN "
+        "( "
+        "SELECT v.LedgerHash "
+        "FROM Validations v "
+        "LEFT OUTER JOIN Ledgers l "
+        "ON v.LedgerHash = l.LedgerHash "
+        "WHERE l.LedgerHash is NULL "
+        "LIMIT %u "
```

Comment:
Two alternatives to using `boost::format`: 1) Just use `std::to_string` for the int. 2) Use `soci::use` For something this simple, I'd just go with `std::to_string`